apps/hash-api/src/graph/knowledge/system-types/notification.ts

```diff
@@ -63,28 +69,16 @@ export const getNotificationFromEntity: PureGraphFunction<
 /**
  * Create a system notification entity.
  *
- * @param params.title - the title of the notification
- *
  * @see {@link createEntity} for the documentation of the remaining parameters
  */
 export const createNotification: ImpureGraphFunction<
```

Comment:
This function isn't used, might be simpler to remove it to avoid having to think about it in future refactors.

It looks like `getNotificationById` and `getNotificationFromEntity` are also unused. `archiveNotification` is in use but could probably just be `updateEntity`. Maybe not this PR but worth doing at some point

Reply:
Fair point, have removed them in https://github.com/hashintel/hash/pull/3551/commits/102fc812394ae9da4b4878348989f297bc4316dd